sql/core/src/main/scala/org/apache/spark/sql/execution/window/WindowExec.scala

```diff
@@ -58,7 +58,7 @@ import org.apache.spark.sql.types.{CalendarIntervalType, DateType, IntegerType,
  *     4. 1 PRECEDING AND 1 FOLLOWING
  *     5. 1 FOLLOWING AND 2 FOLLOWING
  * - Offset frame: The frame consist of one row, which is an offset number of rows away from the
- *   current row. Only [[OffsetWindowFunction]]s can be processed in an offset frame.
+ *   current row. Only [[FrameLessOffsetWindowFunction]]s can be processed in an offset frame.
```

Comment:
how about `UnboundedOffsetWindowFunctionFrame`? Is it also a "Offset frame"?

Reply:
I will update the doc.